### DISCUSSION TITLE
Cypress Test Runner doesn't load changes

### DISCUSSION THREAD

**Discussion by Tchougbe** [cypress-io/cypress#15909] [Component Testing]
Hi , 
I'm newbie to Cypress and have some weird behavior with Test runner. 
Here is my code: 
```
describe('My First Test', () => {  
      
    it('finds the content "je demarre"', () => {
      cy.on('uncaught:exception', (err, runnable) => {
              return false
      })
    
      cy.visit('https://rutabago.com/');
        //  cy.contains('Je démarre');
    })
})
```
But  when i change website url to visit (https://preprod.rutabago.com/) , Test runner never takes the changes nd still trying to visit the first website link. 
Did i have some cache to clear?

**Comment by lmiller1990** [comment]
Are you using the component testing runner (`yarn cypress open-ct`) or e2e (`yarn cypress open`)?

**Comment by scapomolina** [comment]
I am dealing with the same problem. Any new change I do to the code, after save the project in VS Code then I run cypress and it does not execute  the new changes in the runner.I execute npm run cypress open per  e2e .

**Reply by lmiller1990** [reply]
Hey, what version of Cypress? Can you share a minimal reproduction by any chance? Happy to help, but I'll need more information.
